components/search-form.tsx

```diff
@@ -161,17 +164,29 @@ export function SearchForm({
                   >
                     <option value="">Semua</option>
                     {buckets.map((bucket: any, bIdx: number) => {
-                      return (
-                        bucket.doc_count > 0 &&
-                        bucket.key && (
+                      if (bucket.key) {
+                        if (checkDocSize && bucket.doc_count > 0) {
+                          return (
+                            <option
+                              key={`option-${key}-${bIdx + 1}`}
+                              value={bucket.key}
+                            >
+                              {bucket.key}
+                            </option>
+                          );
+                        }
+
+                        // FAQ page doesn't check the doc_count
+                        return (
                           <option
                             key={`option-${key}-${bIdx + 1}`}
                             value={bucket.key}
                           >
                             {bucket.key}
                           </option>
-                        )
-                      );
+                        );
+                      }
```

Comment:
For this part, buckets that have `doc_count` 0 is still printed even though the checkDocSize is true because it goes to the 2nd return. So now, all the filters are always shown.

If the intention for FAQ page is to always show all the categories, another way to do it is to generate `filterItems` value from groupped `faq` instead of using value generated from `useSearch`. 
This way, the filter category order can be in the same order as how the FAQ items are shown which probably is more natural for FAQ page.
![image](https://user-images.githubusercontent.com/9606523/126347443-6ddca256-da87-45db-bfe8-12ae4a3a1205.png)

Sorry, it's a closed pull request but I just have time to check it.

Reply:
Could you please report another issue for this Mbak @ekamuktia? So we can do a rework later. Thanks! 🙏

Reply:
Okay~!

Reply:
https://github.com/kawalcovid19/wargabantuwarga.com/pull/229

Reply:
For option no 2, I think it will need more time to be done, since it need to refactor the way we select the selected filter.